lib/dont_pick_my_number.dart

```diff
@@ -1,3 +1,46 @@
-int calculate() {
-  return 6 * 7;
+import 'dart:io';
+import 'dart:math';
+
+void main() {
+  int num;                        // 뽑을 숫자
+  Set<int> todayNumber = {};      // 오늘 뽑힌 숫자 저장
+  List<int> nobody = [1, 5, 10, 14, 19, 25];    // 없는 숫자
+  List<int> result = [];          // 회차별 뽑힌 숫자 저장할 결과리스트
+
+  // 어제 뽑힌 번호는 제외시키기 위해 불러온다.
+  final file = File('lib/save.txt');
+  List<String> yesterdayList = file     // 파일에 저장된 문자열을 가공하여 List<String> 타입으로 변환
+      .readAsStringSync()
+      .replaceAll('{', '')
+      .replaceAll('}', '')
+      .split(', ');
+  Set<int> yesterdayNumber = yesterdayList.map((e) => int.parse(e)).toSet(); //List 안의 String을 int로 변환
+  print('어제 뽑혔던 번호는 $yesterdayNumber 입니다.');
+
+  for (int i = 1; i <= 2; i++) {        // i 회차 만큼 반복한다.
+    Set<int> set = {};                  // 중복 제거할 컬렉션
+    while (result.length < 5) {         // 결과값이 5개가 될 때까지 반복 시작
+      num = Random().nextInt(38) + 1;   // 1부터 39 사이 랜덤한 숫자 하나를 뽑는다.
```

Comment:
```suggestion
     final num = Random().nextInt(38) + 1;   // 1부터 39 사이 랜덤한 숫자 하나를 뽑는다.
```
이렇게 사용하시면 위에서 num을 따로 선언해주지 않아도 될 것 같습니다!